src/otx/algo/common/losses/iou_loss.py

```diff
@@ -76,6 +79,96 @@ def forward(
         )
 
 
+class IoULoss(nn.Module):
```

Comment:
Do we use losses from this module for other tasks except for Object Detection?

Reply:
No, but I just moved it into common/iou_loss because it looked strange that there are two files (common and detection) about iou loss.

Reply:
I think we could consider moving this loss to the Detection folder instead.

Reply:
I'll revert it in the next PR about yolov7 integration.